### PR TITLE
[FCE-478] Rename mobile test app

### PR DESCRIPTION
**Pull request by MiloszFilimowski**
## Description

Renamed example app from `video-chat` to to `fishjam-chat`

## Motivation and Context

Use the same name for the example as in the [web-client-sdk](https://github.com/fishjam-cloud/web-client-sdk)

## How has this been tested?

- Built and run both Android and iOS
- Run e2e tests. Tests failed locally, but built and run successfully. Created new [issue to fix](https://linear.app/swmansion/issue/FCE-481/fix-local-e2e-example-tests)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/fishjam-chat/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "video-chat-example",
+  "name": "fishjam-chat-example",
```
Comment (mironiasty) [inline examples/fishjam-chat/package.json:2]: Let's use similar approach as in `web-client-sdk` and have:
```suggestion
  "name": "@fishjam-example/fishjam-chat",
```

**Review by mironiasty — APPROVED** [review]
Maybe let's do it in separate PR - but I think it would be also worth to update app name from FishjamExample to FishjamChat
@karkakol WDYT?

**Comment by MiloszFilimowski** [comment]
> Maybe let's do it in separate PR - but I think it would be also worth to update app name from FishjamExample to FishjamChat @karkakol WDYT?

I thought about this but decided it was too much effort compared to what's gained, as native renames usually are difficult to perform.

My proposition is after we bump the Expo version, we could recreate the example app and migrate the UI. It will be easier to start fresh and we would have a clean example project with proper names everywhere.

File: examples/fishjam-chat/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "video-chat-example",
+  "name": "@fishjam-example/fishjam-chat",
```
Comment (karkakol) [inline examples/fishjam-chat/package.json:2]: Would not it be better to just name it `fishjam-chat`?
It is shorter, easier to write and looks more natural imho.

**Review by karkakol — APPROVED** [review]
image

File: examples/fishjam-chat/package.json (hunk 2 above)
Comment (MiloszFilimowski) [reply examples/fishjam-chat/package.json:2]: As @mironiasty has suggested, this will now be consistent with the web SDK.

**Pull request merged**